core/src/main/java/org/apache/accumulo/core/fate/AbstractFateStore.java

```diff
@@ -57,8 +57,8 @@
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
 // TODO 4131 should probably add support to AbstractFateStore, MetaFateStore,
-// and UserFateStore to accept null lockID and zooCache (maybe make these fields
-// Optional<>). This could replace the current createDummyLockID(). This support
+// and UserFateStore to accept null lockID (maybe make this field Optional<>).
```

Comment:
When I have a PR w/ a lot of TODOs sometimes I will try to open an issue for each TODO and remove it before merging.  Could do that for this PR but do not have to, if not doing it  then it would be good to use a more unique TODO which I think is being done here?  Do all of the TODOs have the 4131 suffix?  If so then it will be easy to find them with grep.  Then we just need an open  issue that records the fact that all `TODO 4131` need to be done.

Reply:
Yes, I have all the TODOs marked `TODO 4131`. I was planning on creating new issues and deleting all the `TODO 4131` when this PR is ready to be merged.